creating_images/guidelines.adoc

```diff
@@ -261,21 +261,28 @@ link:s2i.html[S2I Requirements] topic.
 [[use-uid]]
 *Support Arbitrary User IDs*
 
-In order to support running containers with volumes mounted in a secure fashion,
-images should be capable of being run as any arbitrary user ID. When
-{product-title} mounts volumes for a container, it configures the volume so it
-can only be written to be a particular user ID, and then runs the image using
-that same user ID. This ensures the volume is only accessible to the appropriate
-container, but requires the image be able to run as an arbitrary user ID.
-
-Running a container with an arbitrary user ID also has the benefit of ensuring
-that a process which is able to escape the container due to a vulnerability in
-the container framework will not have specific user permissions on the host
-system.
-
-To accomplish this, directories that must be written to by processes in the
-image should be world-writable. In addition, the processes running in the
-container must not listen on privileged ports (ports below 1024).
+To provide additional security against processes escaping the container due
+to a container engine vulnerability and thereby achieving escalated permissions 
+on the host node, {product-title} runs containers using an arbitrarily assigned uid by default.
+
+For an image to support running as an arbitray user, directories and files that may be 
+written to by processes in the image should be owned by the root group and read/writable 
+by that group.  Files to be executed should also have group execute permissions.  The following
+Dockerfile fragment will set permissions for `/some/directory` and all child directories:
```

Comment:
We try to avoid "this WILL do that".  Suggest: "The following Dockerfile fragment sets permissions..."